src-tauri/src/main.rs

```diff
@@ -303,3 +305,12 @@ fn disable_all_but_core_caller(game_install: GameInstall) -> Result<(), String>
 async fn get_installed_mods_caller(game_install: GameInstall) -> Result<Vec<NorthstarMod>, String> {
     get_installed_mods(game_install)
 }
+
+#[tauri::command]
+/// Installs the specified mod
+async fn install_mod_caller(
```

Comment:
Not sure if that's actually necessary... For one it needs to compare versions cause we definitely wanna allow install for newer versions of the same mod as well as maybe a force-reinstall in cases where the user may have changed the mod files and wants to force-reinstall.

I feel like it might make more sense to handle this in the frontend in the sense that it will have a button that shows `Install`, `Force reinstall`, or `Update` depending on (the existence of) the locally installed mod.

Reply:
Fair!